tests/unit_tests/scheduler/test_lsf_driver.py

```diff
@@ -327,6 +327,15 @@ def test_parse_bjobs(job_id, username, job_state):
     }
 
 
+def test_parse_bjobs_handles_output_with_exec_host_split_over_two_lines():
```

Comment:
Has it happened that the output was split over two lines?

Reply:
Yes.